.github/workflows/main.yml

```diff
@@ -77,16 +78,16 @@ jobs:
         pip install cmake==3.18.4
         echo "/Library/Frameworks/Python.framework/Versions/2.7/bin" >> $GITHUB_PATH
         brew install openmpi
+        brew reinstall gcc
     - name: info
       run: |
         export OMPI_CXX=g++-9
+        export OMPI_CC=gcc-9
+        export OMPI_FC=gfortran-9
         mpicxx -v
         cmake --version
     - name: build
       run: |
-        export OMPI_CXX=g++-9
-        export OMPI_CC=gcc-9
-        export OMPI_FC=gfortran-9
```

Comment:
this doesn't work, I think. You can check the output: you are now using clang instead.

Reply:
This is still there but only shifted upwards, to see the output in the runners.

Reply:
see https://github.com/dealii/candi/pull/249/checks?check_run_id=3174164517#step:5:3159 (this should be gcc...)

Reply:
Thanks for this detailed look! Initially I shifted the export command in the previous step to output the compiler versions and thought that the variables live until the end of this job. Anyway now it is updated and should use the correct compilers. However, what do you think about shifting to version 10 or even 11 of gcc suite?

Reply:
the current default of homebrew is gcc@11 and it is the recommended version for homebrew openmpi.

But, deal.II v9.2 is incompatible with gcc@11.